packages/csv-parser-order/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@commercetools/csv-parser-order",
+  "version": "1.0.1",
+  "description": "Module that parses order csv to json",
+  "keywords": [
+    "commercetools",
+    "csv",
+    "nodejs",
+    "parser",
+    "orders",
+    "json"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs/tree/master/packages/csv-parser-order"
+  },
+  "homepage": "https://github.com/commercetools/nodejs/tree/master/packages/csv-parser-order#readme",
+  "bugs": "https://github.com/commercetools/nodejs/issues",
+  "license": "MIT",
+  "contributors": [
+    "Abimbola Idowu <abimbola.idowu@commercetools.com>",
+    "Selwyn Versteeg <selwyn.versteeg@commercetools.com> (https://selwyn.cc/)",
+    "Jan Juna <jan.juna@commercetools.com> (https://janjuna.cz/)"
+  ],
+  "main": "./lib/index.js",
+  "bin": {
+    "csvparserorder": "bin/csvparserorder.js"
+  },
+  "files": [
+    "bin",
+    "lib"
+  ],
+  "dependencies": {
+    "JSONStream": "^1.3.1",
+    "csv-parser": "^1.11.0",
+    "highland": "^2.10.5",
+    "lodash": "^4.17.4",
+    "npmlog": "^4.0.2",
+    "object-path": "^0.11.4",
+    "pretty-error": "^2.0.3",
+    "yargs": "^8.0.1",
+    "sinon": "^2.3.2",
```

Comment:
I put them to `dependencies` because of the linter errors:
```
  packages/csv-parser-order/test/line-item-state.spec.js:3:1
  ✖    3:1   streamtest should be listed in the project's dependencies, not devDependencies.  import/no-extraneous-dependencies

  packages/csv-parser-order/test/add-return-info.spec.js:2:1
  ✖    2:1   streamtest should be listed in the project's dependencies, not devDependencies.  import/no-extraneous-dependencies

  packages/csv-parser-order/test/deliveries.spec.js:1:1
  ✖    1:1   streamtest should be listed in the project's dependencies, not devDependencies.  import/no-extraneous-dependencies
  ✖    3:1   sinon should be listed in the project's dependencies, not devDependencies.       import/no-extraneous-dependencies
```